paasta_tools/chronos_tools.py

```diff
@@ -394,6 +395,15 @@ def get_healthcheck_mode(self, _):
         # Healthchecks are not supported yet in chronos
         return None
 
+    def get_desired_state_human(self):
```

Comment:
I don't think this should be an instance level function, just a module function that takes a state and returns a human readable version (so the get_desired_state is moved to the caller). Same with the version in marathon tools. I know this change is like for like as is, but still, worth cleaning up if you're touching the code. To do this you'll need to edit ``chronos_serviceinit`` + ``marathon_serviceinit`` to change the caller to call the right function.